src/fields/Field.ts

```diff
@@ -586,6 +586,9 @@ export function FieldValueManager<O extends BaseOptions, F extends Constructor<I
             this.target = target
             this.eF = existingField
             this.value = this.eF?.value
+            if (!this.eF) {
+                
+            }
```

Comment:
What's this used for?

Reply:
probably just a bad rebase/cleanup on my part!